packages/camp/package.py

```diff
@@ -137,6 +137,7 @@ class Camp(CMakePackage, CudaPackage, ROCmPackage):
     maintainers = ["trws"]
 
     version("main", branch="main", submodules="False")
+    version("2023.06.0", sha256="b828f650727717578317d7e048048b279a2670bb82a0f0aaf3196fefb0bafca9")
```

Comment:
which commit does this sha hash refer to? There is a camp tag v2023.06.0.

Reply:
It is the commit of the release. I was following the pattern in the file.

Reply:
Not sure why it is using commit hashes, each camp release is tagged.